source/sixtrack.f90

```diff
@@ -702,7 +702,7 @@ subroutine daten
 
   case("DIST") ! Beam Distribution
     if(openBlock) then
-      continue
+      dist_enable = .true.
```

Comment:
(picky) shouldn't this flag be set to `.true.` only if there is a file to read?

Reply:
The idea is that the presence of the DIST block should trigger the call to dist_readDist, and the parsing will fail there if the file is non-existent (which includes the case of a missing READ flag).